build.gradle

```diff
@@ -26,9 +26,10 @@ dependencies {
         exclude(group: "net.fabricmc.fabric-api")
     }
     // ModMenu
-    modCompileOnly modRuntimeOnly("com.terraformersmc:modmenu:${project.mod_menu_version}") {
-        exclude(group: "net.fabricmc.fabric-api")
-    }
+//    modCompileOnly modRuntimeOnly("com.terraformersmc:modmenu:${project.mod_menu_version}") {
+//        exclude(group: "net.fabricmc.fabric-api")
+//    }
+	modApi "com.terraformersmc:modmenu:6.2.1"
```

Comment:
Not a fan of a fixed version here

Reply:
Mb, this is my first time modding Minecraft. So was I supposed to do it like

build.gradle
```
    modCompileOnly modRuntimeOnly("com.terraformersmc:modmenu:${project.mod_menu_version}") {
        exclude(group: "net.fabricmc.fabric-api")
    }
```
gradle.properties
```
# Dependencies
	fabric_version=0.80.0+1.19.4
	cloth_config_version=10.0.96
	mod_menu_version=6.2.1
```
?

Reply:
Yes exactly :)
Otherwise it looks good